tests/analytics.bats

```diff
@@ -0,0 +1,141 @@
+setup() {
+  export SOLR_COLLECTION=gxa-analytics-v1
+}
+
+@test "Check that curl is in the path" {
+    run which curl
+    [ "$status" -eq 0 ]
+}
+
+@test "Check that awk is in the path" {
+    run which awk
+    [ "$status" -eq 0 ]
+}
+
+@test "Check that jq is in the path" {
+    run which jq
+    [ "$status" -eq 0 ]
+}
+
+@test "Check that sdrf converter is in the path" {
+    run which condSdrf2tsvForGXAJSONFactorsIndex.sh
+    [ "$status" -eq 0 ]
+}
+
+@test "Check that filtering script is in the path" {
+    run which jsonFilterEmptyFields.sh
+    [ "$status" -eq 0 ]
+}
+
+@test "Check valid json output from sdrf converter" {
+    condSdrf2tsvForgxaJSONFactorsIndex.sh $BATS_TEST_DIRNAME/example-conds-sdrf.tsv | jq -s .
+    [  $? -eq 0 ]
+}
+
+@test "Check that filtering doesn't remove any cell IDs" {
```

Comment:
I'm concerned that this test is actually working. Can you check that any schema or test file hasn't been dragged from single cell that would make this test pass please? Thanks.

Reply:
We should still be checking the filtering, just not with cell_ids.

Reply:
ahh, this is probably passing because count is 0 at both sides. Can you please choose a meaningful field for this check for bulk? Thanks.